package.json

```diff
@@ -24,93 +24,105 @@
   "license": "Apache-2.0",
   "main": "build/index.js",
   "dependencies": {
+    "@babel/runtime-corejs3": "~7.14.5",
     "@date-io/luxon": "~1.3.13",
-    "@fortawesome/fontawesome-svg-core": "~1.2.34",
-    "@fortawesome/pro-duotone-svg-icons": "~5.15.2",
-    "@fortawesome/pro-light-svg-icons": "~5.15.2",
-    "@fortawesome/pro-regular-svg-icons": "~5.15.2",
-    "@fortawesome/pro-solid-svg-icons": "~5.15.2",
-    "@fortawesome/react-fontawesome": "~0.1.0",
     "@material-ui/core": "~4.11.3",
     "@material-ui/lab": "~4.0.0-alpha.57",
     "@material-ui/pickers": "~3.2.10",
     "@material-ui/styles": "~4.11.3",
     "clsx": "~1.1.1",
-    "immutable": "4.0.0-rc.9",
-    "lodash": "~4.17.20",
-    "luxon": "~1.25.0",
     "polished": "~4.1.0",
     "prop-types": "~15.7.2",
     "react-scrolllock": "~5.0.1",
-    "react-select": "~4.0.2",
-    "react-transition-group": "~4.4.1",
-    "redux-reqseq": "0.5.0"
+    "react-select": "~4.3.1",
+    "react-transition-group": "~4.4.2",
+    "redux-reqseq": "0.6.1"
   },
   "devDependencies": {
-    "@babel/cli": "~7.13.10",
-    "@babel/core": "~7.13.10",
-    "@babel/plugin-proposal-class-properties": "~7.12.13",
-    "@babel/plugin-proposal-object-rest-spread": "~7.12.13",
-    "@babel/plugin-transform-runtime": "~7.12.15",
-    "@babel/polyfill": "~7.12.1",
-    "@babel/preset-env": "~7.13.10",
-    "@babel/preset-flow": "~7.12.13",
-    "@babel/preset-react": "~7.12.13",
-    "@babel/register": "~7.12.13",
-    "@babel/runtime": "~7.13.10",
-    "@storybook/addon-actions": "~6.1.17",
-    "@storybook/addon-docs": "~6.1.17",
-    "@storybook/addon-storysource": "~6.1.17",
-    "@storybook/addons": "~6.1.17",
-    "@storybook/react": "~6.1.17",
-    "@storybook/source-loader": "~6.1.17",
-    "@storybook/storybook-deployer": "~2.8.7",
-    "@storybook/theming": "~6.1.17",
+    "@babel/cli": "~7.14.5",
+    "@babel/core": "~7.14.5",
+    "@babel/plugin-proposal-class-properties": "~7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "~7.14.5",
+    "@babel/plugin-transform-runtime": "~7.14.5",
+    "@babel/preset-env": "~7.14.5",
+    "@babel/preset-flow": "~7.14.5",
+    "@babel/preset-react": "~7.14.5",
+    "@babel/runtime": "~7.14.5",
```

Comment:
I don't think this is necessary because of `@babel/runtime-corejs3` in deps above.